spec/factories.rb

```diff
@@ -17,6 +17,7 @@
       other_language_details { Faker::Lorem.paragraph_by_chars(number: 200) }
       further_information { Faker::Lorem.paragraph_by_chars(number: 300) }
       uk_residency_status { 'I have the right to study and/or work in the UK' }
+      disclose_disability { true }
```

Comment:
To avoid tests relying on static data in the factories:
 
```suggestion
      disclose_disability { [true, false].sample }
```